identity/src/main/java/com/stripe/android/identity/camera/IdentityAggregator.kt

```diff
@@ -50,12 +84,28 @@ internal class IdentityAggregator(
     ): Pair<InterimResult, FinalResult?> {
         if (isFirstResultReceived) {
             val previousState = state
-            state = previousState.consumeTransition(result)
+            state = previousState.consumeTransition(frame, result)
             val interimResult = InterimResult(state)
-            return if (state.isFinal) {
-                interimResult to FinalResult(frame, result, state)
-            } else {
-                interimResult to null
+            return when (state) {
```

Comment:
optional: you could `return interimResult to when (state) {`

Reply:
done